bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/TargetEnvironment.java

```diff
@@ -235,8 +241,9 @@ public boolean containsTarget(SPTarget target) {
      * Returns <code>true</code> if the given target is the base position in
      * this environment.
      */
+    @Deprecated
     public boolean isBasePosition(SPTarget target) {
-        return target == base;
+        return target == getBase();
```

Comment:
Yeah I haven't figured out a fix for that one yet. It's not clear whether it's sufficient to say `allSpTargets.exists(_ == target)` or not. All the call sites come from code I don't understand.